### PR TITLE
Feature/index external links

### PR DESCRIPTION
**Pull request by kmcelwee**
* Here's the layout I landed on, I assume we'd want to iterate on this?
Screen Shot 2020-10-15 at 11 24 57 AM
* Do we want the external link icon? (the box with an arrow to the outside)
* Do we have a preference between null values and empty strings in Solr? Some tags are empty strings, and `link_s` is sometimes an empty string, but with my new `iiif_link_s` field, I set it to `None` so it doesn't appear in all solr records.
* The image server at Cambridge is currently down, so links are difficult to test right now.

### CODE REVIEW

File: scripts/templates/results.html [hunk 1]
```diff
@@ -26,8 +26,14 @@
...
             <div class="fl w-80 pa2">
                 <span>{{ result.description_txt[0]|safe }}</span><br/>
-                <!-- source mss -->
-                <span class="id fl purple avenir">{{ result.shelfmark_s }} | {{ result.library_s }}</span><br>
+                <div class="w-100 flex justify-between">
+                    <!-- source mss -->
+                    <div class="id fl purple avenir">{{ result.shelfmark_s }} | {{ result.library_s }}</div><br>
+                    <!-- View external source -->
+                    {% if result.link_s %}
+                    <div><a class="id no-underline fl purple avenir" href="{{ result.link_s }}">View external record</a></div>
```
Comment (thatbudakguy) [inline scripts/templates/results.html:34]: any reason for the `no-underline`? usually color alone isn't sufficient to indicate a link to all users.

**Review by thatbudakguy — APPROVED** [review]
idk if we need the external link icon now, but we should definitely make a note so @gissoo can either choose an appropriate icon or design one if none fit our purposes. I'd say `None` is preferable wherever possible, but defer to @rlskoeser.

**Comment by rlskoeser** [comment]
How did I not know that sending a value of `None` in the dict results in Solr not storing a value? I could swear that a previous system I used stored that as the string "None" 😬 . Yes, let's use `None` instead of empty string for both links and iiif urls.

This display seems fine to me. While we're prototyping, let's keep the visuals as plain and simple as possible. No icons, minimal visuals. I don't want us to unintentionally make any visual decisions or assumptions that we'll have to try to forget!

**Comment by rlskoeser** [comment]
@kmcelwee not specific to this PR, but is the library always repeated in the shelfmark? If so, let's drop it.

File: scripts/index.py [hunk 2]
```diff
@@ -46,9 +46,14 @@ def index():
         'library_s': row['Library'],
         'shelfmark_s': row['Shelfmark - Current'],
         'shelfmark_txt': row['Shelfmark - Current'],
-        'tags_txt': [tag.strip() for tag in row['Tags'].split('#')],
-        'tags_ss': [tag.strip() for tag in row['Tags'].split('#')],
+        'tags_txt': [tag.strip() for tag in row['Tags'].split('#') if tag.strip() != ''],
```
Comment (rlskoeser) [inline scripts/index.py:49]: Thanks for catching this, had just noticed it myself. Just `if tag.strip()` is sufficient.

File: scripts/templates/results.html (hunk 1 above)
Comment (rlskoeser) [reply scripts/templates/results.html:34]: agree, let's drop the `no-underline`

**Review by rlskoeser — APPROVED** [review]

**Comment by kmcelwee** [comment]
@rlskoeser The shelfmarks do not necessarily map to libraries in a consistent way, for example:
```
PGPID: 19961
T-S 12.399 | CUL

PGPID: 30854
JRL GENIZAH FRAGMENT ARABIC 202 | Rylands
```

I didn't have time to replace empty strings with None. Should we make a chore?

I think this should be merged? We haven't finished QA for #7, but as @rlskoeser said, it's probably useful for them to have tags while testing.

**Pull request merged**